### DISCUSSION TITLE
Arm64 Support

### DISCUSSION THREAD

**Discussion by DK1DA** [calcom/docker#332] [General]
Hey everyone, first of all thanks for the docker support! I'm gradually migrating all my server infrastructure to arm64 and I've noticed that this repo is not yet available on linux for arm64, are there plans to add support?

```
no matching manifest for linux/arm64/v8 in the manifest list entries
```

If not I'd be willing to help, but generally speaking, when looking at the dependencies, I think this should just be a step in your build pipeline and work out of the box.

Cheers,
Marcus

**Comment by keithwillcode** [comment]
There are no immediate plans, no. Would love the help!
